src/App.tsx

```diff
@@ -20,12 +32,15 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                onChange={(event) => {
+                  handleChange(event);
+                }}
```

Comment:
let's use your new `handleChange` function
```suggestion
                onChange={handleChange}
```